.eslintrc.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: "module",
+  },
+  env: {
+    node: true,
+    es6: true,
+  },
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+  plugins: ["prettier"],
+  extends: ["eslint:recommended", "prettier"],
+  rules: {
+    "prettier/prettier": "error",
+  },
+  overrides: [
+    {
+      files: ["**/*.{ts,tsx}"],
+      parser: "@typescript-eslint/parser",
+      parserOptions: {
+        project: "tsconfig.json",
+      },
+      env: {
+        browser: true,
+        node: true,
+      },
+      plugins: [
+        "@typescript-eslint",
+        "jsx-a11y",
+        "simple-import-sort",
+        "react",
+      ],
+      extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
+        "plugin:react-hooks/recommended",
+        "plugin:jsx-a11y/recommended",
```

Comment:
I have a feeling that some of these are already included in `eslint-config-next`?

Reply:
I have usually started from minimally modified version of what Next.js offers. Something as simple as this https://github.com/wunderio/client-fi-nelonen-site/blob/445837094be30b0253328b914deaad406d5b247c/.eslintrc.json

Reply:
> I have a feeling that some of these are already included in eslint-config-next?

Hmm, true. All those plugins are mentioned at https://nextjs.org/docs/basic-features/eslint. I removed them in https://github.com/wunderio/nextjs-project/pull/217/commits/aa164fc0d9b2fc3233636287be40fc1d51ca04c9